frontend/src/app/testResults/uploads/CsvSchemaDocumentation.tsx

```diff
@@ -172,6 +172,25 @@ const CsvSchemaDocumentation = () => {
             How to format and upload a CSV file to report test results in bulk
             through SimpleReport.
           </p>
+          <p>
+            SimpleReport’s bulk results uploader lets you report multiple test
+            results at once using a CSV file. When you submit your results, the
+            uploader tool verifies the data, then sends your results to the
+            relevant state and/or local health department(s).
+            <br />
+            <br />
+            The results uploader is connected with all{" "}
+            <a
+              href="/getting-started/organizations-and-testing-facilities/where-does-simplereport-work"
+              className="usa-link"
+            >
+              states partnered with SimpleReport.
+            </a>{" "}
```

Comment:
```suggestion
            The results uploader is connected with all{" "}
            <a
              href="/getting-started/organizations-and-testing-facilities/where-does-simplereport-work"
              className="usa-link"
            >
              states partnered with SimpleReport
            </a>{" "}.
```
Removing period from link to follow content best practices